### ISSUE TITLE
pxp:set-base-uri doesn't emit document content

### ISSUE THREAD

**Issue by phtyson** [ndw/xmlcalabash1#162]
In version 1.0.18-95, pxp:set-base-uri does not put document content on result port.

```
  <pxp:set-base-uri uri="http://example.org/primary">
     <p:input port="source">
       <p:inline>
      <doc><title>Document 2</title><p>paragraph</p></doc>
       </p:inline>
     </p:input>
   </pxp:set-base-uri>

  <p:store href="primary.xml"/>
```

produces a document with empty content:

```
 <doc xmlns:c="http://www.w3.org/ns/xproc-step" 
    xmlns:pxp="http://exproc.org/proposed/steps"/>
```

(It does, however, seem to set the base URI of the document.)

**Comment by ndw** [comment]
It's hard to know what the problem is without the examples. You can fence code blocks on github with three back quotes on a single line before and after the example.

**Comment by phtyson** [comment]
On Fri, 2014-08-15 at 15:48 -0700, Norman Walsh wrote:

> It's hard to know what the problem is without the examples. You can
> fence code blocks on github with three back quotes on a single line
> before and after the example.

Edited comment to add example. Empty document appears on result port of
pxp:set-base-uri. Expecting copy of input source document.

Thanks,
--Paul

**Comment by ndw** [comment]
I believe this is fixed in XML Calabash 1.0.19, released today.
